src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -5029,15 +4910,11 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
 
         static EnablePSRemotingCommand()
         {
-            string workflowConfigXml = string.Format(CultureInfo.InvariantCulture, _workflowConfigXml,
-                string.Format(CultureInfo.InvariantCulture, "{0}.{1}", PSVersionInfo.PSVersion.Major, PSVersionInfo.PSVersion.Minor),
-                PSSessionConfigurationCommandBase.GetLocalSddl());
-
             string enableRemotingScript = string.Format(CultureInfo.InvariantCulture,
                 enableRemotingSbFormat, PSSessionConfigurationCommandUtilities.GetWinrmPluginShellName(),
                 // Workflow endpoint configuration will be done through Register-PSSessionConfiguration
```

Comment:
Please remove the // Workflow... comment.

Reply:
Done.